### PR TITLE
added growing tiles

### PR DESCRIPTION
**Pull request by cawolf**

### CODE REVIEW

**Review by MollocH — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -108,6 +108,8 @@ The payload needs to be a JSON object with a given set of keys:
 | `priority` | `integer` | defines the display size of the tile on the board: the higher the _priority_, the bigger the tile in relation to other tiles |
 | `date` | `date` | defines when the monitoring data was created. Serves as starting point of the `idle` calculation. |
 | `path` | `string` | optionally defines a tree path to place the monitoring at. Tree paths are formatted like `rootName.branchName.leafName`, see [tree paths](#tree-layout) for more information. |
+| `tileExpansionIntervalCount` | `integer` | optionally defines the number of `idleTimeoutInSeconds` intervals after an `error` tile will grow by `tileExpansionGrowthExpression`. Defaults internally to `1`, see [error tile growth](#error-tile-growth) for more information. |
```
Comment (MollocH) [inline README.md]: optionally defines the number of times `idleTimeoutInSeconds` has to be elapsed after which an `error` tile will grow by `tileExpansionGrowthExpression`. Defaults to `1` if not provided. See [error tile growth](#error-tile-growth) for more information.

File: README.md [hunk 2]
```diff
@@ -108,6 +108,8 @@ The payload needs to be a JSON object with a given set of keys:
...
+| `tileExpansionGrowthExpression` | `string` | optionally defines a growth expression for the growth of `error` tiles. Allowed formats are `+ <positive integer>` or `* <positive integer>`. Defaults internally to `+ 1`, see [error tile growth](#error-tile-growth) for more information. |
```
Comment (MollocH) [inline README.md]: optionally defines an expression for the growth of `error` tiles. Allowed formats are `+ <positive integer>` and `* <positive integer>`. Defaults to `+1` if not provided. See [error tile growth](#error-tile-growth) for more information.

File: README.md [hunk 3]
```diff
@@ -213,6 +215,22 @@ Additionally, you will notice the URL changing when you navigate the tree. You c
 
 Please keep in mind, that you should *not* declare a tree path both as branch node and leaf node. Pushing data into a branch node will result in validation errors and will not be processed.
 
+### Error tile growth
+
+From our experience, low prioritized monitorings usually won't be fixed unless there is nothing else to do, and this happens not often. Therefore, we thought of a way to bring these monitorings to our attention.
```
Comment (MollocH) [inline README.md]: From our experience, low prioritized monitorings usually won't be fixed unless there is nothing else to do. And let's be honest, this does not happen very often.

File: README.md [hunk 4]
```diff
@@ -213,6 +215,22 @@ Additionally, you will notice the URL changing when you navigate the tree. You c
...
+
+Every tile, which at some point switches to `error` will grow over time in priority (and on screen), so that it can compete with high priority monitorings. Each tile will grow each
```
Comment (MollocH) [inline README.md]: Willst du nicht eher sagen: jede Kachel die derzeit im Error Status ist ?

File: README.md (hunk 2 above)
Comment (cawolf) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (cawolf) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (cawolf) [reply README.md]: Also:
Each `error` tile ...
?

**Pull request merged**